Task2/style/styles.css

```diff
@@ -0,0 +1,11 @@
+.circle{
+    background: blue;
+    border-radius: 50%;
+    width: 250px;
+    height: 250px;
+    position: fixed; 
+    top: 50%;
+    left: 50%;
+    margin-top: -125px;
```

Comment:
try to remove 
    margin-top: -125px;
    margin-left: -125px;
    and solve it in dynamic way

Reply:
https://github.com/RahafSalah/FrontEndTraining/pull/4